PolyEngine/Scripts/ProjectTool.py

```diff
@@ -0,0 +1,123 @@
+import argparse
+import os
+import shutil
+import sys
+import fileinput
+from enum import Enum
+
+class ActionType(Enum):
+    CREATE = 'Create',
+    UPDATE = 'Update'
+
+    def __str__(self):
+        return self.name
+
+# Custom action for project creation
+class CreateProjectAction(argparse.Action):
+    def __init__(self, option_strings, dest, nargs, **kwargs):
+        if nargs is not 2:
+            raise ValueError("Nargs must be 2 !")
+        super(CreateProjectAction, self).__init__(option_strings, dest, nargs, **kwargs)
+    def __call__(self, parser, namespace, values, option_string=None):
+        setattr(namespace, self.dest, ActionType.CREATE)
+        setattr(namespace, self.metavar[0], values[0])
+        setattr(namespace, self.metavar[1], values[1])
+
+class UpdateProjectAction(argparse.Action):
+    def __init__(self, option_strings, dest, nargs, **kwargs):
+        if nargs is not 1:
+            raise ValueError("Nargs must be 1 !")
+        super(UpdateProjectAction, self).__init__(option_strings, dest, nargs, **kwargs)
+    def __call__(self, parser, namespace, values, option_string=None):
+        setattr(namespace, self.dest, ActionType.UPDATE)
+        setattr(namespace, self.metavar, values[0])
+
+# Functions
+# Create project function
+
+# Setup:
+# $PROJECT_NAME$ - name of the project
+# $PROJECT_PATH$ - relative path of the project (game) directory (relative to root project dir)
+# $GAME_CLASS_NAME$ - name of the game class
+# $ENGINE_DIR$ - absolute path to engine root directory
+
+def ReplaceTagsInFile(fileToSearch, tagsAndValues):
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#naming-conventions), function and variable names should be written in snake_case